apps/api/src/domain/models/context/edge-connection.from-event-history.spec.ts

```diff
@@ -161,7 +212,7 @@ describe(`EdgeConnection.fromEventHistory`, () => {
 
                 const { text, languageCode } = edgeConnection.note.getOriginalTextItem();
 
-                expect(text).toBe(noteText);
+                expect(text).toBe(noteTranslation);
 
                 expect(languageCode).toBe(languageCode);
 
```

Comment:
We will want to make sure we have two test cases:
- one for a self note (created by `CREATE_NOTE_ABOUT_RESOURCE`)
- one for a connecting note (created by `CONNECT_RESOURCES_WITH_NOTE`)